Ecommerce connector (#406)
akeneo/pim-api-docs: content/guides/ecommerce-connection/step4-define-your-first-scope.md

# Define the scope of your first version

## Which version / edition of the PIM should my connector be “compatible” with?

As [mentioned before](step2-understand-akeneo-pim.html), there are [several editions](https://www.akeneo.com/compare-editions/) and several supported version of Akeneo PIM.

> Of course of the PIM version/edition depends on your Business but we encourage you to develop a connector that is fully compatible with our last **Serenity edition**.

Reviewer: ```suggestion
Of course of the PIM version/edition depends on your business but we encourage you to develop a connector that is fully compatible with our last **Serenity edition**.
```

> Of course of the PIM version/edition depends on your business but we encourage you to develop a connector that is fully compatible with our last **Serenity edition**.

Reviewer 2: ```suggestion
Of course, the choice of the PIM version/edition depends on your business but we strongly encourage you to develop a connector that is fully compatible with our latest **Serenity edition**.
```

Please, [start here](https://api.akeneo.com/getting-started/connect-the-pim-4x/welcome.html) to connect the PIM with the API!

## Which eCommerce solution versions should my connector be compatible with?

It depends on your eCommerce solution and your business strategy.

> You have to choose these versions accordingly:

Reviewer 2: ```suggestion
You have to choose these versions according to:
```

> * The number of potential users that have or will have this version.

Reviewer 2: ```suggestion
* The number of potential users owning this version.
```

> * The lifetime of this version (it is useless to specifically develop a version compatible with a version of your eCommerce solution if this one doesn't exist anymore in a month or two).

Reviewer 2: ```suggestion
* The lifetime of this version. It is useless to specifically develop a version compatible with a version of your eCommerce solution if this one won't exist anymore in a couple of months.
```

> ## What are Akeneo's recommendations regarding the scope of an initial eCommerce connector release?

Reviewer 2: ```suggestion
## What are Akeneo's recommendations to build the first iteration of an eCommerce connector release?
```
